.github/workflows/tools.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/abel
     - name: ABEL Tests
-      if: steps.changes.outputs.abel == 'true'
+      if: steps.changes.outputs.abel == 'true' OR steps.changes.outputs.instance_validator == 'true' OR steps.changes.outputs.type_validator == 'true'
```

Comment:
the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

Reply:
Done.